### PR TITLE
Sanitize URL if the integration is ER type

### PR DESCRIPTION
**Pull request by vgarcia13**
- Change scheme from `http` to `https`
- Add port to base_url if present

If the site is ER, the `base_url` will only contain scheme,  hostname and port (to avoid any path present)

### CODE REVIEW

File: cdip_admin/integrations/models/v1/models.py [hunk 1]
```diff
@@ -230,6 +232,20 @@ def _pre_save(self, *args, **kwargs):
...
+            # Cleanup
+            url_parse = urlparse(self.endpoint)
+
+            scheme = url_parse.scheme
+            if scheme == "http":
+                scheme = "https"
+
+            port = ""
+            if url_parse.port:
+                port = f":{url_parse.port}"
```
[the remark this answers is not in the document]
Comment (vgarcia13) [reply cdip_admin/integrations/models/v1/models.py:245]: Not 100% sure but it is unlikely

[the remark this answers is not in the document]
Comment (chrisdoehring) [reply cdip_admin/integrations/models/v1/models.py:245]: Good question. I would be happy with just using the `netloc` property from the parsed URL. That way we don't need to worry about whether a port is included.

Ex. 
```python
from urllib.parse import urlparse

urlparse('https://somesite.pamdas.org:8443/api/v1.0/subjects')
ParseResult(scheme='https', netloc='somesite.pamdas.org:8443', path='/api/v1.0/subjects', params='', query='', fragment='')
```

[the remark this answers is not in the document]
Comment (chrisdoehring) [reply cdip_admin/integrations/models/v1/models.py:245]: So then below you would say:
```python
    self.endpoint = f"{u.scheme}://{u.netloc/"

...
    subject_url = urljoin(self.endpoint, '/api/v1.0/subject')
```

And it's worth the time to learn exactly how `urljoin` behaves. For instance, how it treats a trailing slash on the base url and how it treats leading slash on the path argument.

File: cdip_admin/integrations/models/v2/models.py [hunk 2]
```diff
@@ -192,6 +192,20 @@ def _pre_save(self, *args, **kwargs):
             if "broker" not in self.additional:
```
Comment (chrisdoehring) [inline cdip_admin/integrations/models/v2/models.py:192]: Shorthand could be beneficial here too. This eliminates the chance that one of the two literal `"broker"` values is accidentally changed.
```python
    self.additional.setdefault('broker', 'gcp_pubsub')
```

Comment (marianobrc) [reply cdip_admin/integrations/models/v2/models.py:192]: great idea, and we can probably remove this setting sometime as the only supported broker is PubSub now

**Review by chrisdoehring — APPROVED** [review]

**Review by marianobrc — APPROVED** [review]

**Pull request merged**